### PR TITLE
[razor-network-voting] update strategy

### PR DESCRIPTION
**Pull request by rajkharvar**
Changes proposed in this pull request:
- Update subgraph endpoint
- Update strategy to fetch score
- Update addresses in `examples.json`

### CODE REVIEW

**Comment by ChaituVR** [comment]
@adi44 Can you plz check and confirm this?

File: src/strategies/razor-network-voting/index.ts [hunk 1]
```diff
@@ -61,38 +62,38 @@ export async function strategy(
     }
   };
 
-  if (snapshot !== 'latest') {
-    // @ts-ignore
-    params.delegators.__args.block = { number: snapshot };
-  }
-
```
Comment (ChaituVR) [inline src/strategies/razor-network-voting/index.ts:68]: This should not be removed @rajkharvar The results should be depended on the block number, else the results can be manuplated

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/razor-network-voting/index.ts (hunk 1 above)
Comment (rajkharvar) [reply src/strategies/razor-network-voting/index.ts:68]: Ok. will update that.

Comment (rajkharvar) [reply src/strategies/razor-network-voting/index.ts]: @ChaituVR I have made the required changes.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**